cli/cmd/start/start.go

```diff
@@ -36,6 +37,25 @@ func StartCmd(cfg *config.Config) *cobra.Command {
 		Args:  cobra.MaximumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			var projectPath string
+			if clean {
+				if len(args) > 0 {
+					err := deleteFilesWithExtensions(args[0], ".db", ".wal")
+					if err != nil {
+						return err
+					}
+				} else {
+					currentDir, err := filepath.Abs("")
+					if err != nil {
+						return err
+					}
+
+					err = deleteFilesWithExtensions(currentDir, ".db", ".wal")
+					if err != nil {
+						return err
+					}
+				}
+			}
```

Comment:
1. You need to push the reset down into the implementation of `local.NewApp` because the correct DB path won't be resolved until then (it becomes relative to `projectPath` in certain cases).
2. Use `drivers.Drop(...)` for dropping the DB